api/health_test.go

```diff
@@ -37,7 +37,7 @@ func TestHealth(t *testing.T) {
 	res, err = server.Client().Get(server.URL + "/health/ready")
 	require.NoError(t, err)
 	defer res.Body.Close()
-	require.Equal(t, http.StatusOK, res.StatusCode)
+	require.Equal(t, http.StatusServiceUnavailable, res.StatusCode)
 	require.NoError(t, json.NewDecoder(res.Body).Decode(&result))
 	assert.Equal(t, "ok", result.Status)
```

Comment:
Shouldn't this not be "ok" if we get `http.StatusServiceUnavailable`?

Reply:
The thing is that at this point of the execution of the test the rules are not yet loaded, so that's why I'm expecting a service unavailable error.
However now that I think of it, I could probably wait loop with a timeout for this check to be `http.StatusOk`

Reply:
Right but what I meant is that the body should not be `{"status":"ok"}` if the status is not ok (because the service is not available). Or am I missing something?

Reply:
Indeed that's weird, I'll check that

Reply:
So I've found why: the variable `result` was still containing the value of the previous test (`/health/alive`).

But in any case I've fix the test, now I init the registry created at the begining of the test file, and therefore I load the rules.
I've also added a sleep of 100ms to be sure that the event had the time to propagate, but thanks to the alive http query it was not necessary (I've still added it to be absolutely sure)